diff_cover/violations_reporter.py

```diff
@@ -116,14 +116,14 @@ def _get_classes(self, xml_document, src_path):
              src_abs_path in [
                  os.path.join(
                      source,
-                     clazz.get('filename')
+                     os.path.normpath(clazz.get('filename').replace("\\", "/"))
                  ) for source in sources]]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_abs_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_abs_path]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_rel_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_rel_path]
```

Comment:
Not sure this is doing what you think it's doing. First it's replacing backslashes with forward slashes, then `os.path.normpath` is normalising them back to backslashes (on Windows). The net result is a Windows-style path. This happens to work with `src_rel_path` because it's ended up Windows-style somewhere else in the code. A safer way to compare paths could be e.g. `os.path.normpath(clazz.get('filename')) == os.path.normpath(src_rel_path)`.